backend/app/api/routes/users.py

```diff
@@ -72,7 +75,11 @@ def create_user(*, session: SessionDep, user_in: UserCreate) -> Any:
     return user
 
 
-@router.patch("/me", response_model=UserPublic)
+@router.patch(
+    "/me",
```

Comment:
I think `/me` route needs special permission. We need to differentiate between users who can manage other users and user's ability to update their information.